operator-common/pom.xml

```diff
@@ -124,6 +124,10 @@
             <groupId>org.apache.kafka</groupId>
             <artifactId>kafka-clients</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
```

Comment:
Why do we need this when you don't seem to be changing anything in `operator-common`? And when exactly the same dependency seems ot be few lines above?

Reply:
I know that this will happen. Actually i do not know, in has some connection with Paolo added lines with `AdminClientInterface`. I can not build project project because of this. Anyway, it could be some inteliJ cache...i will try without it and again build.

Reply:
I added the `kafka-clients` dependency in one of my previous PR when I moved the `AdminClientProvider` and default implementation to operator commen for being used in UO (other than CO). It needs `Admin` interface from that package for example.